server/container/util/src/test/java/org/apache/james/util/ReactorUtilsTest.java

```diff
@@ -80,31 +80,22 @@ public Integer getCounter() {
 
     @Nested
     class ToInputStream {
-        @Test
-        void shouldConsumeNothingWhenNoBytesAreRead() {
```

Comment:
This test may no longer hold because of prefetch but it is still IMO valuable.

Reply:
I do not see how you to express it then